src/Rest/SkosResponse.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
The interface seems to be unused, not sure about it's use either?

Reply:
Yep that was just an sketch to distinguish OpenSKOS responses among others. Not sure if it is really needed now. So it is up to discussion.